uber/site_sections/preregistration.py

```diff
@@ -236,19 +249,16 @@ def prereg_payment(self, session, payment_id, stripeToken):
         raise HTTPRedirect('paid_preregistrations?payment_received={}', charge.dollar_amount)
 
     def paid_preregistrations(self, session, payment_received=None):
-        if not self.paid_preregs:
-            raise HTTPRedirect('index')
-        else:
-            preregs = [session.merge(Charge.from_sessionized(d)) for d in self.paid_preregs]
-            for prereg in preregs:
-                try:
-                    session.refresh(prereg)
-                except:
-                    pass  # this badge must have subsequently been transferred or deleted
-            return {
-                'preregs': preregs,
-                'total_cost': payment_received
-            }
+        preregs = [session.merge(Charge.from_sessionized(d)) for d in self.paid_preregs]
+        for prereg in preregs:
+            try:
+                session.refresh(prereg)
+            except:
+                pass  # this badge must have subsequently been transferred or deleted
+        return {
+            'preregs': preregs,
+            'total_cost': payment_received
+        }
```

Comment:
I don't fully understand the change here. Is this fixing something? I feel like this should be reverted. Am I wrong?